tests/Geppetto/README.md

```diff
@@ -0,0 +1,55 @@
+[![Prestashop](https://i.imgur.com/qDqeQ1E.png)](https://www.prestashop.com)
```

Comment:
Can you rename the directory in lowercase please `gepetto` or better `e2e`.

Reply:
In fact I've just respected the previous convention, E2E directory has already named with capital letters any suggestion @eternoendless ?